### PR TITLE
update release notes + bump version for EasyBuild v4.9.2

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -6,10 +6,172 @@ search:
...
 including 40 different (compiler) toolchains.
-It contains 255 software-specific easyblocks and 43 generic easyblocks,
-alongside 19,487 easyconfig files.
+It contains 259 software-specific easyblocks and 43 generic easyblocks,
+alongside 19,985 easyconfig files.
+
+
+## EasyBuild v4.9.2 (12 June 2024) {: #release_notes_eb492 }
+
+bugix/update release
```
Comment (bedroge) [inline docs/release-notes.md:17]:
```suggestion
bugfix/update release

```

Comment (bedroge) [reply docs/release-notes.md:17]: The same typo is also in the release notes of the 4.9.1, 4.8.2, and 4.8.1.

File: docs/release-notes.md [hunk 2]
```diff
@@ -6,10 +6,172 @@ search:
...
+    - also consider `$CRAY_PE_LIBSCI_PREFIX_DIR` to determine installation prefix for `cray-libsci` ([#4551](https://github.com/easybuilders/easybuild-framework/pull/4551))
+    - symlink downloaded repo at specified commit when using `--from-commit` so easyconfigs for dependencies are found ([#4552](https://github.com/easybuilders/easybuild-framework/pull/4552))
+- other changes:
+    - code cleanup in `easyblock.py` ([#4519](https://github.com/easybuilders/easybuild-framework/pull/4519))
+    - stop running unit tests on Python 3.5 ([#4530](https://github.com/easybuilders/easybuild-framework/pull/4530))
+
+**easyblocks**
+
+- 4 new software-specific easyblocks: QuantumESPRESSO with CMake ([#3257](https://github.com/easybuilders/easybuild-easyblocks/pull/3257)), AEDT ([#3281](https://github.com/easybuilders/easybuild-easyblocks/pull/3281)), optiSLang ([#3300](https://github.com/easybuilders/easybuild-easyblocks/pull/3300)), MetalWalls ([#3311](https://github.com/easybuilders/easybuild-easyblocks/pull/3311), [#3331](https://github.com/easybuilders/easybuild-easyblocks/pull/3331))
```
Comment (bedroge) [inline docs/release-notes.md:39]: The metalwalls URL doesn't look right.

File: docs/release-notes.md [hunk 3]
```diff
@@ -6,10 +6,172 @@ search:
...
+    XZ 5.4.5, xorg-macros 1.20.1, xprop 1.2.7, xtb 6.7.0, xxd 9.1.0307, yaml-cpp 0.8.0, zarr 2.17.1, zfp 1.0.1,
+    zlib-ng 2.1.6, zlib 1.3.1, zstd 1.5.6
+- minor enhancements, including:
+    - add missing (optional) dependency pyproject-metadata to scikit-build-core ([#20391](https://github.com/easybuilders/easybuild-easyconfigs/pull/20391))
+    - add `hatch-requirements-txt` extension to hatchling easyconfigs ([#20389](https://github.com/easybuilders/easybuild-easyconfigs/pull/20389))
+    - install pkg-config files for ncurses 6.4 when using `GCCcore` toolchain ([#20405](https://github.com/easybuilders/easybuild-easyconfigs/pull/20405))
+    - use regular `configure` instead of wrapper script for recent UCX easyconfigs ([#20428](https://github.com/easybuilders/easybuild-easyconfigs/pull/20428))
+    - add RISC-V support to UCX 1.15.0 ([#20429](https://github.com/easybuilders/easybuild-easyconfigs/pull/20429)), UCC 1.2.0 ([#20432](https://github.com/easybuilders/easybuild-easyconfigs/pull/20432)), BLIS 0.9.0 ([#20468](https://github.com/easybuilders/easybuild-easyconfigs/pull/20468)), PAPI 7.1.0 (20659)
```
Comment (bedroge) [inline docs/release-notes.md:117]: Link for PAPI is missing.

Comment (bedroge) [reply docs/release-notes.md:117]:
```suggestion
    - add RISC-V support to UCX 1.15.0 ([#20429](https://github.com/easybuilders/easybuild-easyconfigs/pull/20429)), UCC 1.2.0 ([#20432](https://github.com/easybuilders/easybuild-easyconfigs/pull/20432)), BLIS 0.9.0 ([#20468](https://github.com/easybuilders/easybuild-easyconfigs/pull/20468)), PAPI 7.1.0 ([#20659](https://github.com/easybuilders/easybuild-easyconfigs/pull/20659))

```

File: docs/release-notes.md (hunk 2 above)
Comment (bedroge) [reply docs/release-notes.md:39]: Actually, it is correct, just two PRs with very similar numbers 😄

File: docs/release-notes.md (hunk 3 above)
Comment (boegel) [reply docs/release-notes.md:117]: fixed with force push in 6ee4943d

File: docs/release-notes.md (hunk 1 above)
Comment (boegel) [reply docs/release-notes.md:17]: fixed with force push in 6ee4943d

**Review by bedroge — APPROVED** [review]

**Pull request merged**